### ISSUE TITLE
ignite run is failing on ssh key gen on FIPS enabled machine

### ISSUE THREAD

**Issue by junaid18183** [weaveworks/ignite#135]
The failure is happening if the host machine is fips enabled 

```
[root@XXXXXX# ignite run weaveworks/ignite-ubuntu     --cpus 2     --memory 1GB     --ssh     --name my-vm
INFO[0000] Docker image "weaveworks/ignite-kernel:4.19.47" not found locally, pulling...
INFO[0003] Imported OCI image "weaveworks/ignite-kernel:4.19.47" (49.0 MB) to kernel image with UID "a81e3c5b1130b712"
INFO[0004] Removed VM with name "my-vm" and ID "b8fd329361ee5432"
command ["ssh-keygen" "-q" "-t" "ed25519" "-N" "" "-f" "/var/lib/firecracker/vm/b8fd329361ee5432/id_b8fd329361ee5432"] exited with "ED25519 keys are not allowed in FIPS mode\r\n": exit status 255
```

**Comment by luxas** [comment]
Thanks for the report. Commented on the PR, we can target this for v0.4.1

**Comment by luxas** [comment]
This is fixed with #136, and will be available in v0.4.1. Thanks :tada:!
